my-app/src/Routes/AuthenticatedRoute.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Game from '../app/screens/Game/index';
+import RegisterFromContainer from '../app/screens/LogIn/index';
+
+const AuthenticatedRounter = props => {
+  return props.logged ? <Game /> : <RegisterFromContainer />;
```

Comment:
Mmmm this will have a problem if multiple routes are supported. I'd use a `Route` here, and receive a `component` from `props` like a common `Route` do.